pyproject.toml

```diff
@@ -11,5 +11,58 @@ exclude = ["venv"]
 "avro/resources/dictionary.py" = ["F601"]
 "tests/test_main.py" = ["F601"]
 
-[tool.ruff.format]
-quote-style = "single"
\ No newline at end of file
+[tool.poetry]
+name = "avro-py"
+version = "2024.5.1"
+description = "A modern Pythonic implementation of Avro Phonetic."
+authors = ["HitBlast <hitblastlive@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "avro"}]
+homepage = "https://pypi.org/project/avro-py"
+repository = "https://github.com/hitblast/avro.py"
+documentation = "https://hitblast.github.io/avro.py"
+keywords = [
+    'python',
+    'phonetics',
+    'avro',
+    'avro phonetic',
+    'bangla',
+    'bengali',
+    'bengali phonetics',
+    'transliteration',
+]
+classifiers=[
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+]
+
+[tool.poetry.scripts]
+avro = "avro.cli:main"
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4.0"
```

Comment:
`<4.0` part is not needed but it's okay.

Reply:
The build fails if the part isn't added.